types/dispatchr/addons/createStore.d.ts

```diff
@@ -0,0 +1,13 @@
+import { StoreClass, Store } from '../index';
+
+interface StoreOptions {
+    initialize?(): void;
+    storeName: string;
+    handlers: { [event: string]: string };
+    [prop: string]: any;
+}
+
+type CreateStore = <This extends StoreOptions>(options: This & ThisType<This & Store>) => StoreClass;
```

Comment:
The result's instance type also extends `Omit<This, 'statics' | 'storeName' | 'handlers' | 'mixins'>`, but I'm not sure it's possible to correctly type this...

mixins are a mistake 😢